docs/developer/surfaces/cli/03-update.md

```
@@ -0,0 +1,151 @@
+
+## Examples
+
+### Basic Usage
+
+```bash
+pls update ./my-project
+```
+
+Output:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)